testharness/nip55queryclient/build.sh

```
@@ -0,0 +1,114 @@
+
+SDK="${ANDROID_SDK_ROOT:-${ANDROID_HOME:-}}"
+if [ -z "$SDK" ]; then
+  echo "error: set ANDROID_SDK_ROOT (or ANDROID_HOME) to your Android SDK dir" >&2
+  exit 1
+fi
+
+shopt -s nullglob
+# Highest-versioned match of a glob, or empty (no failing `ls` in a pipe, so a missing
+# dir does not trip `set -o pipefail` before the friendly guards below can report it).
+pick_latest() {
+  local matches=($1)
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)